src/build/core.js

```diff
@@ -1,18 +1,19 @@
 (function(root, factory) {
 
   if (typeof define === 'function' && define.amd) {
-    define(['backbone', 'underscore','backbone.radio', 'backbone.babysitter'],
+    define(['backbone', 'underscore','backbone.radio', 'backbone.babysitter','backbone-metal'],
       function(Backbone, _) {
-      return (root.Marionette = root.Mn = factory(root, Backbone, _));
+      return (root.Marionette = root.Mn = factory(root, Backbone, _, Metal));
     });
   } else if (typeof exports !== 'undefined') {
     var Backbone = require('backbone');
     var _ = require('underscore');
     var BabySitter = require('backbone.babysitter');
     var Radio = require('backbone.radio');
-    module.exports = factory(root, Backbone, _);
+    var Metal = require('backbone-metal');
+    module.exports = factory(root, Backbone, _, Metal);
   } else {
-    root.Marionette = root.Mn = factory(root, root.Backbone, root._);
+    root.Marionette = root.Mn = factory(root, root.Backbone, root._, root.Metal);
```

Comment:
I'm not sure root.Metal is necessary?  I assume it'll be available via Backbone.Metal by this point..  does metal even export a global?  For instance if you `<script>` in Radio it'll be available on `Backbone.Radio`, but not `root.Radio` by default.